tests/parsers/test.manifestjson.js

```diff
@@ -739,5 +739,26 @@ describe('ManifestJSONParser', () => {
         description: 'Icon could not be found at "icons/icon-64.png".',
       });
     });
+
+    it('adds an error if the icon does not have a valid extension', () => {
+      const addonLinter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({
+        icons: {
+          32: 'icons/icon-32.txt',
+          64: 'icons/icon-64.html',
```

Comment:
Let's add one or two valid icons so that we can verify we don't match types we're not supposed to match.